Extension/package.json

```diff
@@ -3286,6 +3286,17 @@
                         "default": false,
                         "markdownDescription": "%c_cpp.configuration.addNodeAddonIncludePaths.markdownDescription%",
                         "scope": "application"
+                    },
+                    "C_Cpp.copilotHover": {
```

Comment:
This is being defined in the "Miscellaneous" section of settings. Just checking to make sure that's what you intended.

![image](https://github.com/user-attachments/assets/118b84e8-8674-49fb-bedc-8d5855ea6929)

Reply:
Unsure if that's the best place for it, but I do acknowledge that it's there for now. I'm open to moving it or making a new Copilot category if folks think that makes sense.